Mathlib/Order/Hom/Basic.lean

```diff
@@ -1171,6 +1171,12 @@ theorem coe_toDualTopEquiv_eq [LE α] :
     (WithBot.toDualTopEquiv : WithBot αᵒᵈ → (WithTop α)ᵒᵈ) = toDual ∘ WithBot.ofDual :=
   funext fun _ => rfl
 
+/-- The coercion `α → WithBot α` bundled as monotone map. -/
+@[simps]
+def coeOrderHom {α : Type*} [Preorder α] : α →o WithBot α where
```

Comment:
Can/should we bundle this as a `↪o` instead?

Reply:
Good point. Done